chaostracing/oltp.py

```diff
@@ -231,11 +231,12 @@ def continuous_hypothesis_iteration(self, iteration_index: int, state: Any) -> N
                     .timestamp()
                     * 1e9
                 )
+                activity_name: str = probe["activity"]["name"]
```

Comment:
good idea on the typing annotation. I rarely type variables as it's fairly obvious but it can help